src/utils.ts

```diff
@@ -62,19 +62,42 @@ export async function removeExportedCredentials(): Promise<string> {
  * of trimmed strings.
  */
 export function explodeStrings(input: string): Array<string> {
-  if (input == null || input.length === 0) {
+  if (!input || input.trim().length === 0) {
     return [];
   }
 
   const list = new Array<string>();
-  for (const line of input.split(`\n`)) {
```

Comment:
This implementation didn't handle escaping strings (where you might actually want a comma or newline in your value)